src/script/lua_api/l_mapgen.cpp

```diff
@@ -754,6 +754,28 @@ int ModApiMapgen::l_get_mapgen_object(lua_State *L)
 }
 
 
+// get_spawn_level(x = num, z = num)
+int ModApiMapgen::l_get_spawn_level(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	int x = luaL_checkinteger(L, 1);
```

Comment:
it's s16 not int

Reply:
Does not matter due implicit conversion in either way.

Reply:
I was copying commom practice in APIs, don't mind either way.

Reply:
Yes much better as it matches the argument for the function, thanks.

Reply:
Done.